src/Tribe/Admin/Move_Tickets.php

```diff
@@ -191,7 +191,7 @@ protected function build_attendee_list( array $attendee_ids, $event_id ) {
 				$this->has_multiple_providers = true;
 			}
 
-			$this->ticket_provider = $provider;
+			$this->ticket_provider = empty( $provider ) && isset( $attendee['provider'] ) ? $attendee['provider'] : $provider;
```

Comment:
Are we favoring `$attendee['provider']` over `$provider`, here? I would think only `isset( $attendee['provider'] )` or `! empty( $provider )` would be appropriate depending on which one we'd rather use.